public/mainnet/migaloo-1/pools_list.json

```diff
@@ -309,45 +309,46 @@
       "rewards_tokens": []
     },
     {
-      "pool_id": "WHALE-ampMNTA",
+      "pool_id": "ampMNTA-WHALE",
       "lpOrder": [
         "WHALE",
```

Comment:
lp order needs to be changed as well

Reply:
When I change the LP order, provide LP doesn't seem to work anymore, will try again in a new commit